### DISCUSSION TITLE
How to use in Android 4.0?

### DISCUSSION THREAD

**Discussion by manyhelp** [SquirrelJME/SquirrelJME#2] [General Questions]
How to use in Android 4.0?

**Comment by xerz-one** [comment]
From what I know, SquirrelJME is not intended to run on Android, which already has its own JVM. However, you may want to try to compile it for Android targets and contribute support for it.

**Comment by XerTheSquirrel** [comment]
Closing this issue. This cannot be used in Android yet, that is pretty much into the future and is not a priority for me to think about right now. However I do plan on Android support.
